models/aeif_cond_beta_multisynapse.cpp

```diff
@@ -67,6 +67,67 @@ RecordablesMap< aeif_cond_beta_multisynapse >::create()
 }
 
 /* ----------------------------------------------------------------
+ * Right-hand side function
+ * ---------------------------------------------------------------- */
+
+extern "C" int
+aeif_cond_beta_multisynapse_dynamics( double,
+  const double y[],
+  double f[],
+  void* pnode )
+{
+  // y[] is the state vector supplied by the integrator,
+  // not the state vector in the node, node.S_.y[].
+
+  typedef nest::aeif_cond_beta_multisynapse::State_ S;
+
+  // get access to node so we can almost work as in a member function
+  assert( pnode );
+  const nest::aeif_cond_beta_multisynapse& node =
+    *( reinterpret_cast< nest::aeif_cond_beta_multisynapse* >( pnode ) );
+
+  const bool is_refractory = node.S_.r_ > 0;
+
+  // Clamp membrane potential to V_reset while refractory, otherwise bound
+  // it to V_peak. Do not use V_.V_peak_ here, since that is set to V_th if
+  // Delta_T == 0.
+  const double& V =
+    is_refractory ? node.P_.V_reset_ : std::min( y[ S::V_M ], node.P_.V_peak_ );
```

Comment:
Since you set dV/dt to zero during the refractory period, is this test really necessary?